app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
It's something I don't remember modifying, will check if it can be removed safely.